docs/reference-guide/10-command_line.md

```diff
@@ -40,6 +40,7 @@ _In all cases, arguments " –h" or "–help" can be used to get help_
 |--no-ts-import | Do not import timeseries into the input folder. <br/> (This option may be useful for running old studies without upgrade)|
 |-m, --mps-export | Export anonymous mps weekly or daily optimal UC+dispatch linear |
 |-s, --named-mps-problems | Export named mps weekly or daily optimal UC+dispatch linear |
+|-o, --solvers-logs | Print solver logs |
```

Comment:
Would prefer "solver-logs" without the plural.

Also, I would remove the short option, I think it's OK to only have the long one.
We can keep the "-o" for a future option that would be more compatible with it.

Reply:
good

Reply:
done!